src/QsCompiler/CompilationManager/CompilationUnitManager.cs

```diff
@@ -862,7 +859,7 @@ T TryQueryFile(FileContentManager f)
         /// Note: this method waits for all currently running or queued tasks to finish
         /// before getting the file content by calling FlushAndExecute.
         /// </summary>
-        public QsFragmentKind[][] GetTokenization(TextDocumentIdentifier textDocument) =>
+        public QsFragmentKind?[][]? GetTokenization(TextDocumentIdentifier textDocument) =>
```

Comment:
I'm curious what this syntax means: `QsFragmentKind?[][]?`

Reply:
```
The outermost array is nullable (GetTokenization may return null).
                   v
QsFragmentKind?[][]?
              ^
The elements in the innermost array are nullable QsFragmentKinds.
```

Reply:
Oh that makes sense. I think my brain was trying to make it more complicated than it is 😁.